parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -25,6 +25,10 @@
 import java.io.Serializable;
 import java.io.UnsupportedEncodingException;
 import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
```

Comment:
Maybe make parquet-column changes as a separate PR? It looks more like a refactoring to me rather than adding support for logical types.